### ISSUE TITLE
Bug: Hostnames of entries in Systems tab are showing up as short name of entries in OSINT tab

### ISSUE THREAD

**Issue by DanFelton** [cyb3rfox/Aurora-Incident-Response#87]
I can't upload the exact example .fox file because it contains actual case data, but I will see if I can reproduce with dummy data.

Essentially when we add systems to the systems tab, entries are appearing in the OSINT tab where the system hostnames are in the short name field. Deleting items from the OSINT tab deletes the corresponding system from the system tab. I'm assuming something about the case file is corrupted but we haven't done anything particular unusual to get to this situation. I will try to reproduce but essentially we've just been adding systems both manually and by importing csv in the system tab. Everything in the system tab worked fine, but the OSINT table seems to have got linked to the system table somehow.

**Comment by cyb3rfox** [comment]
Yes, that's a known bug. I'll fix this one but most of the work now goes into the new version
